api-scripts/Inventory_Hierarchy/ItemsAndHoldings_POST.jmx

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.1.1 r1855137">
```

Comment:
In other jmx, we are using JMeter 5.2.1? Is it possible to upgrade to be consistent with other tests?